src/pages/HomePage.tsx

```diff
@@ -1,32 +1,25 @@
-import { useSelector, useDispatch } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
-import { RootState, increment, decrement } from '../store';
+import { Button } from '../components/Button.tsx'
 
-import reactLogo from '../assets/react.svg';
-import viteLogo from '/vite.svg';
+import pic from '../assets/homepage_main-pic.png'
+import logo from '../assets/LOGO_big.svg'
 
 export const HomePage = () => {
-  const count = useSelector((state: RootState) => state.counter.value);
-  const dispatch = useDispatch();
-  const { t } = useTranslation();
+  const { t } = useTranslation()
 
   return (
     <>
-      <div className="flex justify-center">
-        <img src={viteLogo} className="logo" alt="Vite logo" />
-        <img src={reactLogo} className="logo react" alt="React logo" />
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => dispatch(decrement())}>-</button>
-        <span>{count}</span>
-        <button onClick={() => dispatch(increment())}>+</button>
-      </div>
-
-      <div>
-        <h2>{t('welcome')}</h2>
-      </div>
+      <main className='mx-auto rounded-[16px] bg-white flex justify-between overflow-hidden drop-shadow-xl '>
+        <div className='m-auto h-[305px] w-[336px] py-10 px-5 flex flex-col justify-between items-center'>
```

Comment:
I suggest to store custom css rules into talwind config variables for their reusability